plaidml/edsl/edsl.h

```diff
@@ -361,6 +362,8 @@ class Tensor {
   TensorLens lens_;
 };
 
+using Tensors = std::vector<Tensor>;
```

Comment:
I'm not sure about this name:  I think it makes the code a bit hard to read since Tensor and Tensors are very similar looking.  Maybe TensorList or TensorVec?